modules/microshift-install-bootc-physically-bound-con.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-bootc-physically-bound.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-install-bootc-physically-bound_{context}"]
+= About physically bound bootc image building
+
+When a bootc image is fully self-contained, everything you need to run workloads is embedded with the bootc image, including {microshift-short} and application container images. The underlying mechanism is to pre-pull physically-bound images during image build and then make them available at runtime.
+
+Because embedded images might change with each system update, you cannot pull the images directory to the default container storage. Additional image stores do not work in this case because these image stores are read-only and intended for static image directories.
```

Comment:
```suggestion
Because embedded images might change with each system update, you cannot pull the images directly to the default container storage. Additional image stores do not work in this case because of the current implementation limits, which would not allow bootc image updates for those container images. 
```